lago/providers/libvirt/vm.py

```diff
@@ -636,8 +641,10 @@ def _libvirt_xml(self):
                         'driver',
                         name='qemu',
                         type=dev_spec['format'],
+                        cache='none',
+                        io='native',
                         discard='unmap',
-                        cache='writeback',
+                        iothread='1',
```

Comment:
This gives me:
```
libvirt.libvirtError: unsupported configuration: IOThreads not available for bus scsi target sda
```

Reply:
Seems that iothread asignment is not supported for virtio-scsi. This is missing also in oVirt VM:

virtio-scsi:

      <driver name='qemu' type='qcow2' cache='none' error_policy='stop' io='threads'/>

virtio:

      <driver name='qemu' type='qcow2' cache='none' error_policy='stop' io='threads' iothread='1'/>

I'll remove this part from the patch, we need to check this with libvirt/qemu folks.